R/add_phylopic.r

```diff
@@ -34,13 +37,17 @@
 #'   should be plotted at once. In this case, any other arguments (except for
 #'   `remove_background`) may also be vectors of values, which will be recycled
 #'   as necessary.
+#'   
+#'   If only `color` is specified (and `fill` is NA), the outline and fill color
```

Comment:
I feel like this should be the default behaviour. I am sure there is a reason why it isn't, but just in case!

Reply:
The reason was because the added outline can make silhouettes look slightly different, so I thought it would be best with a transparent outline and the black fill. I've rewritten all of the defaults to use a black color and NA fill as suggested.